pkg/cmd/server/origin/run_components.go

```diff
@@ -404,13 +405,21 @@ func (c *MasterConfig) RunServiceServingCertController(client *kclientset.Client
 // RunImageImportController starts the image import trigger controller process.
 func (c *MasterConfig) RunImageImportController() {
 	osclient := c.ImageImportControllerClient()
-	importRate := float32(c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute) / float32(time.Minute/time.Second)
-	importBurst := c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute * 2
+
+	var limiter flowcontrol.RateLimiter = nil
+	if c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute == -1 {
+		limiter = flowcontrol.NewTokenBucketRateLimiter(math.MaxFloat32, math.MaxInt32)
```

Comment:
THere's an unlimited rate limiter (which is a hardcoded no-op to succeed).  Use that instead.  Also, you should have <=0, because someone could call this code with 0.